compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrTypeConverter.kt

```diff
@@ -135,14 +135,12 @@ class Fir2IrTypeConverter(
             is ConeCapturedType -> {
                 val cached = capturedTypeCache[this]
                 if (cached == null) {
-                    val irType = lowerType?.toIrType(typeContext) ?: run {
```

Comment:
Hi @mglukhikh , I am not confident about this change here. Without it, the logic would convert a `Captured<in String>` to a `String`, which seems wrong. It should instead be converted to `Any` instead. I thought about this and it seems to be the case for `out` projection as well. Hence it appears `lowerType` is useless.